### ISSUE TITLE
Update `Top Up` page to include Utorg and NEARPay fiat onramps

### ISSUE THREAD

**Issue by corwinharrell** [near/near-wallet#2445]
## Summary

This is a design update for the `Top Up` page accessed from the wallet dashboard. With new fiat onramps becoming available, we need a more scalable way to display multiple options from within the wallet.

This issue addresses both #2403 and #2365.

### Approach to Displaying Multiple Options

For now, we will keep the display of fiat onramp options lightweight. Because each of these onramp options is so new, there isn't quite enough data to give us an accurate picture about which ones users prefer, and which see the most volume transacted. 

Paradox of choice is a real problem in the context of UX. Users presented with too many options when faced with a decision will report a negative experience. Intentionally limiting options is often beneficial for the user since it reduces cognitive load and allows them to complete their task more quickly. In particular instances such as during onboarding, we may elect to "feature" a particular onramp to reduce cognitive load. We reserve the right to determine which onramps to feature at particular points in the experience, depending on factors like fees, volume of $NEAR transacted, etc. Naturally and over time, the highest performing and most dependable onramps will be preferred and/or emphasized.

## Resources

Figma Document: https://www.figma.com/file/rraJCzjeTmAXZaDRA5Y1Kq/Top-Up-Page?node-id=108%3A552

<img width="1434" alt="Screen Shot 2022-02-07 at 11 21 32 AM" src="https://user-images.githubusercontent.com/5934188/152828498-d7c9eb98-1340-4519-b81e-7f78d5b20f99.png">

**Comment by stefanopepe** [comment]
I would disable or "grey out" the Rainbow Bridge if the user doesn't have already a minimum balance to pay for gas fees and sign transactions/swaps, keeping the accent to onramps and exchanges first. A balance under 0.5 $near might be too small to cover transaction fees and the increased storage needs (login + fungible tokens state).

**Comment by corwinharrell** [comment]
@stefanopepe I don't think we should be disabling/enabling external links based on a user's balance. That is a non-standard UX practice. Typically an "insufficient" balance error should be reserved and be displayed in context. In the case of the rainbow bridge, this would actually be in the rainbow bridge UI. On this page, we are simply showing what options are available for all users, and they still need to be able to navigate to each of these services and platforms in order to learn about them and decide if they do in fact want to use them.

**Comment by ivan-ilin-nearpay** [comment]
Hi, guys! @corwinharrell @stefanopepe, any news on this issue?
It is assigned for a sprint that has ended 8 days ago. Can we help somehow?

**Comment by agileurbanite** [comment]
@Patrick1904 can you share details on the blockers for this issue?

**Comment by agileurbanite** [comment]
#2419 only addresses the NEARPay integration of this issue and doesn't address Utorg.

**Comment by Patrick1904** [comment]
> @Patrick1904 can you share details on the blockers for this issue?

I was hoping that the [NearPay integration](https://github.com/near/near-wallet/pull/2419#issuecomment-1063522536) could make these UI changes to the 'Top Up' page at the same time. However, that is not a blocker and we can make the UI updates to the 'Top Up' page before any additional fiat onramp integrations are completed.

**Comment by Nakameowdough** [comment]
Can we please add the FTX options to the wallet for other places to purchase NEAR.
